### ISSUE TITLE
[css-forms] Standardize rendering to allow per option styling

### ISSUE THREAD

**Issue by Nokel81** [w3c/csswg-drafts#1685]
Currently (tested in FireFox and in Chrome) you can only format things like font and not much else. It would be nice to be able to style each item individually with all of css.

I would propose that the they be rendered in a form that blocks overflow and onclick the size is created to the bottom is extended and the options animate down. I realize that `select` is on of the things are explicitly rendered very much differently on mobile (or at least on Chrome for Android). This currently even ignores `background-color` from the select. Either these alternative renderings should also have to follow the styling or there should be an easy way to style each one individually. Maybe a `:mobile` selector.

**Comment by Nokel81** [comment]
Since this seems to have been the form since the beginning, is there a reason why `select`s were singled out as the soul HTML element that is not fully syllable?

**Comment by SebastianZ** [comment]
I have a somewhat related request asking to [allow other elements within `<option>` elements](https://discourse.wicg.io/t/support-elements-in-option-elements/2293).

Sebastian

**Comment by Nokel81** [comment]
Yes that looks like a very good idea as well. I think that this would have to also cover how such changes would work on mobile.

I think that things like `<b>` and `<i>` should be just rendered and so could `<img>`s. As for CSS like background colour, font should also be supported. I think that other things like `:hover` should just be ignored by mobile browsers

**Comment by nt1m** [comment]
I think customizable select solves this.
